packages/dev/src/cloudflare-api.js

```diff
@@ -347,11 +352,12 @@ async function createNamespace(title, args) {
 }
 
 /**
- * @param {{ site: StaticSite | null; name: string; accountId: string; accountEmail: string; apiKey: string; }} args
+ * @param {{ site: StaticSite | null; name: string; accountId: string; accountEmail: string | undefined; apiKey: string; }} args
  * @returns {Promise<{ bindings: any[]; kvCleanup: () => Promise<void>; manifest?: string }>}
  */
 async function getSiteBindings(args) {
-  if (!args.site) {
+  const site = args.site;
+  if (!site) {
```

Comment:
I had to add this because TS was complaining about `StaticSite | null`. Not sure if that error would break the tests, though.

Reply:
This looks a bit weird, to be honest. According to https://stackoverflow.com/questions/28975896/is-there-a-way-to-check-for-both-null-and-undefined , it seems it would have been enough to replace `if (!var)` with `if (var != null)` to satisfy your linter.

Reply:
That didn't fix it but I've reverted it anyway. The problem is with some local TypeScript installation I have so it shouldn't affect the CI.